crates/wasm/wit/entity.wit

```diff
@@ -24,6 +24,22 @@ default interface entity {
         apply-base-pose: bool,
     }
 
+    /// Animation clip data
+    record animation-clip {
+        /// BinderId for the animation tracks
+        binders: list<string>,
+        /// Duration in seconds
+        duration: float32,
+        /// If the asset is loaded (duration will be 0 and binders empty if false)
+        loaded: bool,
+        /// Error that occured while fetching asset
+        error: string,
+    }
+
+    set-animation-blend: func(entity: entity-id, weights: list<float32>, time: list<float32>, absolute-time: bool)
+    has-animation-clip: func(clip-url: string) -> bool
```

Comment:
`has-animation-clip` sounds like it exists at all; maybe `animation-clip-loaded`?

Reply:
Added status and moved the api to asset.wit